File: CalcLibrary/src/main/java/com/knowledgelens/calclibrary/KLCalculator.java
package com.knowledgelens.calclibrary;

public class KLCalculator {
    public static int plus(int a, int b){
        return a+b;
    }

    public static int minus(int a, int b){
        return a-b;
    }

    public static int multiply(int a, int b){
        return a*b;
    }

    public static int divide(int a, int b){
        if(b==0){
            throw new IllegalArgumentException("Divide by zero");
        }
        else{
            return a/b;
        }
    }
}

File: CalcLibrary/src/main/java/com/knowledgelens/calclibrary/KLCalculator.kt
package com.knowledgelens.calclibrary

import java.lang.IllegalArgumentException

class KlCalculator {
    companion object{
        fun plus(a:Int,  b:Int):Int{
            return a+b
        }

        fun minus(a:Int,  b:Int):Int{
            return a-b
        }

        fun product(a:Int,  b:Int):Int{
            return a*b
        }

        fun division(a:Int,  b:Int):Int{
            if(b==0){
                throw IllegalArgumentException("Div by zero error")
            }
            else{
                return a/b
            }
        }
    }
}